mvlearn/embed/splitae.py

```diff
@@ -175,111 +188,127 @@ def fit(self, Xs, validation_Xs=None):
+        ), """can't train for negative amount of
             times"""
         device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
 
         view1 = torch.FloatTensor(Xs[0])
         view2 = torch.FloatTensor(Xs[1])
 
-        self.view1Encoder_ = _FullyConnectedNet(view1.shape[1],
-                                                self.hiddenSize,
-                                                self.numHiddenLayers,
-                                                self.embedSize).to(device)
-        self.view1Decoder_ = _FullyConnectedNet(self.embedSize,
-                                                self.hiddenSize,
-                                                self.numHiddenLayers,
-                                                view1.shape[1]).to(device)
-        self.view2Decoder_ = _FullyConnectedNet(self.embedSize,
-                                                self.hiddenSize,
-                                                self.numHiddenLayers,
-                                                view2.shape[1]).to(device)
-
-        self.view1_encoder_ = self.view1Encoder_
-        self.view1_decoder_ = self.view1Decoder_
-        self.view2_decoder_ = self.view2Decoder_
-
-        if self.printGraph:
-            print("Parameter counts: \nview1Encoder: {:,}\nview1Decoder: {:,}"
-                  "\nview2Decoder: {:,}"
-                  .format(self.view1Encoder_.paramCount(),
-                          self.view1Decoder_.paramCount(),
-                          self.view2Decoder_.paramCount())
-                  )
-
-        parameters = [self.view1Encoder_.parameters(),
-                      self.view1Decoder_.parameters(),
-                      self.view2Decoder_.parameters()]
-        optim = torch.optim.Adam(itertools.chain(*parameters),
-                                 lr=self.learningRate)
-        nSamples = view1.shape[0]
-        epochTrainErrors = []
-        epochTestErrors = []
-
-        for epoch in tqdm.tqdm(range(self.trainingEpochs),
-                               disable=(not self.printInfo)):
-            batchErrors = []
-            for batchNum in range(nSamples // self.batchSize):
+        self.view1_encoder_ = _FullyConnectedNet(
+            view1.shape[1], self.hidden_size, self.num_hidden_layers, self.embed_size
+        ).to(device)
+        self.view1_decoder_ = _FullyConnectedNet(
+            self.embed_size, self.hidden_size, self.num_hidden_layers, view1.shape[1]
+        ).to(device)
+        self.view2_decoder_ = _FullyConnectedNet(
+            self.embed_size, self.hidden_size, self.num_hidden_layers, view2.shape[1]
+        ).to(device)
+
+        self.view1_encoder_ = self.view1_encoder_
+        self.view1_decoder_ = self.view1_decoder_
+        self.view2_decoder_ = self.view2_decoder_
+
+        if self.print_graph:
+            print(
+                "Parameter counts: \nview1Encoder: {:,}\nview1Decoder: {:,}"
+                "\nview2Decoder: {:,}".format(
```

Comment:
Can you also change this text to snake case, like "view1_encoder" rather than view1Encoder?

Reply:
Yeah sure, Thanks for pointing out.